### PR TITLE
refactor: #159 change styles for header

### PR DESCRIPTION
**Pull request by MaratReactJs**

### CODE REVIEW

File: mw-webapp/src/styles/_variables.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@ $link-active: rgb(255, 149, 0);
 
 // Header
 $header-background: rgb(85, 122, 149);
+$header-padding: 0 1em;
```
Comment (Howl404) [inline mw-webapp/src/styles/_variables.scss:13]: Why padding? It's looks uncentered

**Review by Howl404 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/styles/_variables.scss (hunk 1 above)
Comment (MaratReactJs) [reply mw-webapp/src/styles/_variables.scss:13]: and how it should be?

Comment (Ekaterina1994) [inline mw-webapp/src/styles/_variables.scss:13]: What do zou think about using rem instead of em?

Comment (Ekaterina1994) [inline mw-webapp/src/styles/_variables.scss:13]: If this padding will be used only in header then need move this var to header.scss. Better skip all styles changes on task #41

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

**Review by Ekaterina1994 — APPROVED** [review]

File: mw-webapp/src/styles/_variables.scss (hunk 1 above)
Comment (MaratReactJs) [reply mw-webapp/src/styles/_variables.scss:13]: I agree, rem is more global

Comment (MaratReactJs) [reply mw-webapp/src/styles/_variables.scss:13]: move all header styles from variables.scss to header.module.scss?

Comment (MaratReactJs) [reply mw-webapp/src/styles/_variables.scss]: removed the extra padding from the title, thanks for noticing

File: mw-webapp/src/component/header/Header.module.scss [hunk 2]
```diff
@@ -5,13 +5,12 @@
...
   width: 100%;
-  padding: $header-padding;
-  background: $header-background;
+  padding: 0 1rem;
+  background: rgb(85, 122, 149);
 }
 
 .title {
   display: flex;
-  padding-top: $title-padding;
```
Comment (Howl404) [inline mw-webapp/src/component/header/Header.module.scss:12]: No no no, title is used in many files, just pass class name from props into title and set padding 0 in it

**Review by Howl404 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/header/Header.module.scss [hunk 3]
```diff
@@ -2,14 +2,15 @@
 
 .header {
   display: flex;
-  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
   width: 100%;
-  background: $header-background;
+  padding: 0 1rem;
```
Comment (tritonJS826) [inline mw-webapp/src/component/header/Header.module.scss:8]: why rem?

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/header/Header.module.scss (hunk 3 above)
Comment (MaratReactJs) [reply mw-webapp/src/component/header/Header.module.scss:8]: At first I thought of using a percentage, but for some reason I chose rem))
it's all about adaptability

**Review by Howl404 — APPROVED** [review]

**Pull request merged**